src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -28,7 +28,8 @@ internal class AzureAppConfigurationProvider : ConfigurationProvider, IConfigura
         private bool _isInitialLoadComplete = false;
         private bool _isFeatureManagementVersionInspected;
         private readonly bool _requestTracingEnabled;
-        private readonly IConfigurationClientManager _configClientManager;
+        private readonly IConfigurationClientManager _refreshConfigClientManager;
+        private readonly IConfigurationClientManager _startupConfigClientManager;
```

Comment:
We might not need two ConfigurationClientManagers since the only configurable startup option should be the overall timeout.

Reply:
I'm thinking the same. Essentially, the startup is like triggering the refresh nonstop (well, 1s interval) until the allocated time runs out. It's the same clients for startup or refresh.

Reply:
Since it looks like we want to remove the startup MaxRetryDelay then I think there's no reason for the 2 managers